tests/system/src/test/java/test/robot/javafx/embed/swing/JFXPanelHiDPITest.java

```diff
@@ -149,7 +151,7 @@ public MyApp() {
         }
 
         private void createScene(final JFXPanel fxPanel) {
-            Platform.runLater(() -> {
+            PlatformImpl.runAndWait(() -> {
```

Comment:
Generally we prefer `test.util.Util::runAndWait` so that exceptions are propagated.

Reply:
good point!